.circleci/deploy-release.yml

```diff
@@ -1,23 +1,31 @@
 version: 2.1
 
-executors:
-  base:
-    macos:
-      xcode: "14.3.1"
-
 parameters:
   deploy-release:
     type: boolean
     default: false
-    
+
 jobs:
   build-testflight-deploy:
-    executor: base
     environment:
       CI: true
+    macos:
+      xcode: "14.3.1"
     resource_class: macos.x86.medium.gen2
     steps:
       - checkout
+      - run:
+          name: Check for MARKETING_VERSION change
+          command: |
+            git fetch origin $CIRCLE_BRANCH
+            git checkout $CIRCLE_BRANCH
+            CURRENT_VERSION=$(grep 'MARKETING_VERSION' Client/Configuration/Common.xcconfig | cut -d ' ' -f3)
```

Comment:
out of curiosity: What's `cut -d ' ' -f3` exactly doing? I wouldn't mind a comment explaining it 😄

Reply:
Thanks for that @ecotopian ! 
Essentially, the `cut -d ' ' -f3` takes the output from `grep` command as input.
For example, let's assume that the `Common.xcconfig` file contains the following line:

`MARKETING_VERSION = 100.2.44`

`grep` will return -> `MARKETING_VERSION = 100.2.44`

The `cut` command will then extract the third field from the input, using a space (' ') as the delimiter.
Output: `100.2.44`

Reply:
Can add a comment to it if you'd prefer 💪

Reply:
Added 😉